### PR TITLE
Update ERC-5564: Move to last call

### PR DESCRIPTION
**Pull request by garyghayrat**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Review by nerolation — APPROVED** [review]

**Comment by garyghayrat** [comment]
Merged in https://github.com/ethereum/ERCs/pull/322#event-12494867858

**Review by moshahin01 — COMMENTED** [review]
Good

File: ERCS/erc-5564.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
 ---
 eip: 5564
 title: Stealth Addresses
-description: Private, non-interactive transfers and interactions
+description: Private, non-interactive transfers and engagements
 author: Toni Wahrstätter (@nerolation), Matt Solomon (@mds1), Ben DiFrancesco (@apbendi), Vitalik Buterin (@vbuterin)
 discussions-to: https://ethereum-magicians.org/t/eip-5566-stealth-addresses-for-smart-contract-wallets/10614
-status: Review
+status: Last Call
+last-call-deadline: 2024-04-30
```
Comment (moshahin01) [inline ERCS/erc-5564.md:8]: Good

Comment (moshahin01) [reply ERCS/erc-5564.md:8]: Good

**Pull request closed without merging**